templates/primary-secondary/primary-secondary.js

```diff
@@ -0,0 +1,67 @@
+import '../../components/colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element';
+
+class TemplatePrimarySecondary extends LitElement {
+
+	static get properties() {
+		return {};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			.container {
+				display: grid;
+				grid-template-columns: 2fr auto 1fr;
+				grid-template-rows: auto 1fr auto;
+				grid-template-areas:
+				"header header header"
+				"primary divider secondary"
+				"footer footer footer";
+				height: 100vh;
+			}
+			.d2l-template-primary-secondary-header {
+				grid-area: header;
+			}
+			.d2l-template-primary-secondary-primary {
+				grid-area: primary;
+				min-width: 320px;
+				overflow-y: auto;
+			}
+			.d2l-template-primary-secondary-divider {
+				grid-area: divider;
+				background-color: var(--d2l-color-mica);
+				width: 1px;
+				height: 100%;

```

Comment:
I think you can use `align-self: stretch` (or `align-items: stretch` if you want ALL the cells to do this) so make the container fill the entire height  & width of the grid cell.

Reply:
Actually `height: 100%`, isn't necessary anymore since I fixed the row to fill up the remaining height on the page. So this divider cell already has the full height.